README.md

```diff
@@ -29,28 +29,28 @@ Follow the [guide](https://meteorclient.com/faq/installation) on the wiki.
 ## Contributions
 We will review and help with all reasonable pull requests as long as the guidelines below are met.
 
-- The license header must be applied to all java source code files.
+- The license header must be applied to all Java source code files.
 - IDE or system-related files should be added to the `.gitignore`, never committed in pull requests.
 - In general, check existing code to make sure your code matches relatively close to the code already in the project.
 - Favour readability over compactness.
 - If you need help, check out the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html) for a reference.
 
 ## Bugs and Suggestions
-Bug reports and suggestions should be made in this repo's [issue tracker](https://github.com/MeteorDevelopment/meteor-client/issues) using the templates provided.  
-Please provide as much information as you can to best help us understand your issue and give a better chance of it being resolved.
+Bug reports and suggestions should be made in this repo's [issue tracker](https://github.com/MeteorDevelopment/meteor-client/issues) using the templates provided.  
+Please provide as much information as you can to best help us understand your issue and give us a better chance of it being resolved.
 
 ## Donations
-All of our work is completely free and non-profit (donations pay only for hosting costs), therefore we are very grateful for all donations made to support us in running our community.  
-Donations can be made via our [website](https://meteorclient.com/donate) and the minimum amount to get donor benefits is €5.  
-You will be rewarded with a role on our Discord server and a customisable in-game cape.  
+All of our work is completely free and non-profit (donations pay only for hosting costs); therefore, we are very grateful for all donations made to support us in running our community.  
+Donations can be made via our [website](https://meteorclient.com/donate) and the minimum amount to get donor benefits is €5.  
+You will be rewarded with a role on our Discord server and a customizable in-game cape.  
```

Comment:
both "customisable" and "customizable" are valid (and i personally prefer the former)